src/StatusQ/Popups/StatusSearchPopup.qml

```diff
@@ -272,6 +289,14 @@ StatusModal {
                         badge.image.source: model.badgeImage
                         badge.icon.isLetterIdenticon: model.isLetterIdenticon
                         badge.icon.color: model.badgeIdenticonColor
+
+                        onItemClicked: {
+                            root.resultItemClicked(itemId)
+                        }
```

Comment:
As mentioned in the other comment, this would be `sensor.onClicked()` and `itemId` should be a property on the component here, inline'd